src/redis_cmd.cc

```diff
@@ -5833,7 +5839,7 @@ class CommandXTrim : public Commander {
 
 CommandAttributes redisCommandTable[] = {
     ADD_CMD("auth", 2, "read-only ok-loading", 0, 0, 0, CommandAuth),
-    ADD_CMD("ping", 1, "read-only", 0, 0, 0, CommandPing),
+    ADD_CMD("ping", -1, "read-only", 0, 0, 0, CommandPing),
```

Comment:
May I ask the reason to make this change? Is there some reference from the Redis document?

The overall change looks good to me. Merging...

Reply:
This digit indicates the number of parameters allowed(including command name), "1 " means there is precisely one, and "-1 "means at least one.  Do you want this [source code](https://github.com/redis/redis/blob/6.2/src/server.c#L715) and the [comment](https://github.com/redis/redis/blob/6.2/src/server.c#L89)?

Reply:
Thank you!